disconnected/mirroring/about-installing-oc-mirror-v2.adoc

```diff
@@ -79,7 +79,7 @@ include::modules/oc-mirror-disk-to-mirror-v2.adoc[leveloffset=+2]
 [id="additional-resources"]
 == Additional resources
```

Comment:
Suggest updating this per latest [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies):
```suggestion
[id="additional-resources_{context}"]
== Additional resources
```

Reply:
I hadn't visited the existing content since this PR was only for restructuring. But I can incorporate this change. Thanks!